Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
I don't understand either what you're saying @kmaglione. The `DOCKER_PREFIX` trick is to allow you to run commands (simply calling `make ...`) inside the container. I'm not sure what `start-docker.sh` has to do with that?